plugins/syncbeacon/plugin.go

```diff
@@ -0,0 +1,69 @@
+package syncbeacon
+
+import (
+	"github.com/iotaledger/goshimmer/packages/shutdown"
+	"github.com/iotaledger/goshimmer/plugins/config"
+	"github.com/iotaledger/goshimmer/plugins/messagelayer"
+	"github.com/iotaledger/hive.go/daemon"
+	"github.com/iotaledger/hive.go/logger"
+	"github.com/iotaledger/hive.go/node"
+	flag "github.com/spf13/pflag"
+	"sync"
+	"time"
+)
+
+const (
+	// PluginName is the plugin name of the sync beacon plugin.
+	PluginName = "Sync Beacon"
+
+	// CfgSyncBeaconBroadcastIntervalSec is the interval in seconds at which the node broadcasts its sync status.
+	CfgSyncBeaconBroadcastIntervalSec = "syncbeacon.broadcastInterval"
+)
+
+func init() {
+	flag.Int(CfgSyncBeaconBroadcastIntervalSec, 30, "the interval at which the node will broadcast ist sync status")
+}
+
+var (
+	// plugin is the plugin instance of the sync beacon plugin.
+	plugin *node.Plugin
+	once   sync.Once
+	log    *logger.Logger
+)
+
+// Plugin gets the plugin instance.
+func Plugin() *node.Plugin {
+	once.Do(func() {
+		plugin = node.NewPlugin(PluginName, node.Disabled, configure, run)
+	})
+	return plugin
+}
+
+// configure events
+func configure(_ *node.Plugin) {
+	log = logger.NewLogger(PluginName)
+}
+
+// broadcastSyncBeaconPayload broadcasts sends a sync beacon to the message layer
+func broadcastSyncBeaconPayload() {
+	syncBeaconPayload := NewSyncBeaconPayload(time.Now().UnixNano())
+	messagelayer.MessageFactory().IssuePayload(syncBeaconPayload)
```

Comment:
It should check for an error while issuing the payload. If there's an error log it as `info`. If successful you can do something like this `log.Infof("issued sync beacon %s", msg.ID())`

Reply:
`messagelayer.MessageFactory().IssuePayload` does not return error. But yeah, we should log the ID